Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -462,7 +470,8 @@ void calcFittestBoundingRegionForLooseTile(
       // We need to compute an accurate bounding region
       result.updatedBoundingVolume = GltfUtilities::computeBoundingRegion(
           model,
-          tileLoadInfo.tileTransform);
+          tileLoadInfo.tileTransform,
+          *result.ellipsoid);
```

Comment:
It's dangerous to dereference this optional here without checking that it's valid. Maybe the field doesn't actually need to be an optional? Or if it does, change to `result.ellipsoid ? *result.ellipsoid : Ellipsoid::WGS84`.

Reply:
Or a third possibility: we're sure the optional has a value in this code path. In which case add an assertion for it. And a comment explaining why it's the case would be helpful would be helpful as well.